src/SprykerEco/Zed/Computop/Business/Hook/Mapper/Init/InitPayuCeeSingleMapper.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Hook\Mapper\Init;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Kernel\Transfer\TransferInterface;
+use SprykerEco\Shared\Computop\ComputopConfig;
+use SprykerEco\Shared\Computop\Config\ComputopApiConfig;
+
+class InitPayuCeeSingleMapper extends AbstractMapper
+{
+    /**
+     * @return string
+     */
+    public function getMethodName(): string
+    {
+        return ComputopConfig::PAYMENT_METHOD_PAYU_CEE_SINGLE;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $computopPaymentTransfer
+     *
+     * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
+     */
+    public function updateComputopPaymentTransfer(
+        QuoteTransfer $quoteTransfer,
+        TransferInterface $computopPaymentTransfer
+    ): TransferInterface {
+        /** @var \Generated\Shared\Transfer\ComputopPayuCeeSinglePaymentTransfer $computopPaymentTransfer */
+        $computopPaymentTransfer = parent::updateComputopPaymentTransfer($quoteTransfer, $computopPaymentTransfer);
+        $computopPaymentTransfer->setMerchantId($this->config->getMerchantId());
```

Comment:
* You can use chained calls because the transfer object implements a `fluent interface` pattern.
* `$computopPaymentTransfer
->setMerchantId()
->setAmount()`.

Reply:
Done